rocketmq-doc/_docs/19-contribute-guide.md

```
@@ -0,0 +1,109 @@
+cargo fmt --all
+```
+
+**Build**
+
+```shell
+cargo build 
+```
+
+**Run Clippy**
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)